experiments/functional/localpv/localpv-provisioning-selected-device/test.yml

```diff
@@ -0,0 +1,130 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+         ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+         ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+
+        - name: Obtain the list of nodes
+          shell: kubectl get nodes --no-headers | grep -v master | awk '{print $1}'
+          args:
+            executable: /bin/bash
+          register: nodes
+
+        - name: Getting the Unclaimed block-device from each node
+          shell: >
+            kubectl get blockdevice -n {{ operator_ns }} -l kubernetes.io/hostname={{ item }}
+            -o jsonpath='{.items[?(@.status.claimState=="Unclaimed")].metadata.name}' | tr " " "\n" | head -n 1
+          register: blockDevice
+          with_items:
+            - "{{ nodes.stdout_lines }}"
+
+        - name: Label the selected block devices
+          shell: >
+            kubectl label bd -n "{{ operator_ns }}" "{{ item.stdout }}" openebs.io/block-device-tag="{{ device_tag }}"
+          args:
+            executable: /bin/bash
+          register: bd_result
+          with_items:
+            - "{{ blockDevice.results }}"
+
+        - name: Forming storage class manifest from template
+          template:
+            src: storage_class.j2
+            dest: storage_class.yml
+
+        - name: Creating storageClass
+          shell: kubectl apply -f storage_class.yml
+          args:
+            executable: /bin/bash
+          register: sc_status
+          failed_when: sc_status.rc != 0
+
+        - name: Forming application manifest from the template
+          template:
+            src: percona.j2
+            dest: percona.yml
+
+        - name: Creating namespace
+          shell: kubectl create ns "{{ namespace }}"
+          args:
+            executable: /bin/bash
+          register: ns_status
+          failed_when: ns_status.rc != 0
+
+        - name: Deploying application
+          shell: kubectl apply -f percona.yml -n "{{ namespace }}"
+          args:
+            executable: /bin/bash
+          register: app_status
+          failed_when: "app_status.rc != 0"
+
```

Comment:
@giri can we check the application status here

Reply:
Done